lib/environment.lua

```diff
@@ -35,6 +36,8 @@ function environment.create(habitat, scriptInstance)
 		new[key] = value
 	end
 
+	new['settings'] = createSettings(habitat)
```

Comment:
We should just use `new.settings` instead of `new['settings']` since there aren't any special characters in this name.